README.md

```diff
@@ -954,42 +954,33 @@ Used for EV chargers.
-in   | cmd.access.set                 | string     |                         | 
-in   | cmd.access.get_report          | null       |                         |
-out  | evt.access.report              | string     |                         |
+in   | cmd.current_session.get_report  | null     |                            | Command for getting energy (kWh) for the current session.
+out  | evt.current_session.report      | float    |                            |
 -|||
-in   | cmd.smart_charge.set           | bool       |                         | true = enable smart charging
-in   | cmd.smart_charge.get_report    | null       |                         | 
-out  | evt.smart_charge.report        | bool       |                         |
+in   | cmd.electric_current.set        | object   | val = {"maxCurrent": 30.0} | Command for setting an electric current maximum amperage.
+in   | cmd.electric_current.get_report | null     |                            |
+out  | evt.electric_current.report     | object   | val = {"maxCurrent": 30.0} |
 -|||
-in   | cmd.rated_current.get_report   | null       |                         | Get the rated or max current for ths CP
-out  | evt.rated_current.report       | int        |                         |
--|||
-in   | cmd.current_session.get_report | null       |                         | Command for getting energy (kWh) for current session.
-out  | evt.current_session.report     | float      |                         |
--|||
-out  | evt.error.report               | string     |                         | 
+out  | evt.error.report                | string   |                            | 
 
 #### Interface props
 
-Name          | Value example                                                 | Description
---------------|---------------------------------------------------------------|-------------
-`sup_states`  | ["disconnected", “requesting”, “charging”, “ready\_to\_charge”, "requesting", "finished", "reserved", "unavailable", "error", "unknown"]  | State of the CP
-`sup_access`  | ["rfid", "open", "app", "local"]                              | Access level for using CP
-`sup_errors`  | []                                                            | Error from CP
-
+Name          | Value example                                                                                                                            | Description
+--------------|------------------------------------------------------------------------------------------------------------------------------------------|-------------
+`sup_states`  | ["disconnected", “requesting”, “charging”, “ready\_to\_charge”, "requesting", "finished", "reserved", "unavailable", "error", "unknown"] | State of the CP.
+`sup_range`   | {"min":7.0, "max":40.0}                                                                                                                  | Optional, supported electric current range. Must be passed if the service supports current management.
```

Comment:
With a range it's usually a good idea to have something like `sup_steps` or `sup_increments` as well. Like, if the range is 7.0 to 40.0, does that mean I can set `7.1`? Or can I only set 7 or 8? Or maybe even only half, like 7, 7.5 and then 8?

Reply:
Outdated - requirements have changed already.